docs/dev/agent_dev_env.md

```diff
@@ -28,11 +33,44 @@ You will also need the Visual Studio for [Visual Studio for Python installer](ht
 
 Download the [gcc toolchain](http://win-builds.org/).
 - From the graphical package manager, select and install the needed libraries, leave the default (select all) if you're unsure.
-- Make sure to select x86_64.
-- Add installation folder to the %PATH%.
+- Make sure to select `x86_64`.
+- Add installation folder to the `%PATH%`.
+
+
+### Python Dependencies
+
+#### Preface
+
+To protect and isolate your system-wide python installation, a python virtual
+environment is _highly_ recommended (though optional). It will help keep a
+self-contained development environment and ensure a clean system Python.
+
+**Please note that due to the [way
+some virtual environments handle executable paths](https://bugs.python.org/issue22213)
+(e.g. `python -m venv`), not all vrtual environment options will be able to run the built
+agent correctly. At this time, the only confirmed virtual enviroment that is known for
+sure to work is `virtualenv`.**
+
+- Install the virtualenv module:
+`pip3 install virtualenv`
```

Comment:
The recommended way of running `pip` is by using the `-m` option (see e.g. [all examples here](https://packaging.python.org/tutorials/installing-packages/))
```suggestion
`python3 -m pip install virtualenv`
```